frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -58,6 +58,7 @@ pods:
           INGEST_ENABLED: false
           ES_JAVA_OPTS: "-Xms{{MASTER_NODE_HEAP_MB}}M -Xmx{{MASTER_NODE_HEAP_MB}}M"
           ELASTIC_VERSION: {{ELASTIC_VERSION}}
+          ELASTIC_STATSD_VERSION: {{ELASTIC_STATSD_VERSION}}
```

Comment:
Q (@benclarkwood) just so that I understand it, an alternative to this would be to include a `TASKCFG_ALL_ELASTIC_STATSD_VERSION` in the marathon `env` section?

Reply:
Correct

Reply:
We've been inconsistent in this